primeui-tester/src/test/java/com/axonivy/ivy/supplements/primeui/tester/TestPrimeUi.java

```diff
@@ -21,20 +21,13 @@
 import com.axonivy.ivy.supplements.primeui.tester.widget.SelectOneMenu;
 import com.axonivy.ivy.supplements.primeui.tester.widget.SelectOneRadio;
 import com.axonivy.ivy.supplements.primeui.tester.widget.Table;
-import com.codeborne.selenide.Configuration;
 
 /**
  * Class to test PrimeUi. Tests on the official Primefaces Showcase.
  */
+@ExtendWith(IvySelenideExtension.class)
```

Comment:
The extension is cool, but I'd make the usage of it even simple and provide an annotation for it 
.... à la @leanProcess in the ivy-core